packages/carbon-web-components/src/components/ui-shell/header-menu.ts

```diff
@@ -24,7 +24,7 @@ const { prefix } = settings;
  * Header menu.
  *
  * @element bx-header-menu
- * @csspart trigger The trigger button.
+ * @csspart topMenuItem The trigger button.
```

Comment:
I think this was referring to `[part="trigger"]`, which hasn't changed, so I don't think we need this change.

Reply:
Oh! I thought it was referring to _trigger, from line 37, which did change?

Reply:
Discussed in a huddle! I made the change and am resolving this